balancer/weightedtarget/weightedtarget_test.go

```diff
@@ -1105,8 +1116,8 @@ func (s) TestBalancerGroup_SubBalancerTurnsConnectingFromTransientFailure(t *tes
 
 	for i := 0; i < 5; i++ {
 		r, err := p.Pick(balancer.PickInfo{})
-		if err == nil || !strings.Contains(err.Error(), scConnErr.Error()) {
-			t.Fatalf("want pick error %q, got result %v, err %q", scConnErr, r, err)
+		if err == nil || !strings.Contains(err.Error(), wantSubConnErr.Error()) {
+			t.Fatalf("want pick error %q, got result %v, err %q", wantSubConnErr, r, err)
```

Comment:
Ignore the pick result and make this consistent with the other similar calls.

```golang
if _, err := p.Pick(balancer.PickInfo{}); err == nil || !strings.Contains(err.Error(), wantSubConnErr.Error()) {...}
```

And got before want in the error message again.